effectful/effectful.cabal

```diff
@@ -84,15 +84,21 @@ library
                      Effectful.Concurrent.STM
                      Effectful.Concurrent.QSem
                      Effectful.Concurrent.QSemN
+                     Effectful.Console.ByteString.Static
+                     Effectful.Console.ByteString.Lazy.Static
                      Effectful.Environment
                      Effectful.FileSystem
                      Effectful.FileSystem.IO
+                     Effectful.FileSystem.IO.ByteString.Static
```

Comment:
I didn't notice this earlier, but the `Static` part is redundant in all newly added modules.

Reply:
Done.